src/dice.class.php

```diff
@@ -129,7 +129,7 @@ function keygen() {
 		$pass = "";
 		for($length = 1; $length < 24; $length++) {
 			$temp = str_shuffle($tempstring);
-			$char = mt_rand(0, strlen($temp));
+			$char = random_int(0, strlen($temp) - 1);
```

Comment:
The second commit changed this to a secure random number generator, because why the heck wasn't a secure RNG used before?!?

Reply:
I'm still trying to figure out why shuffling the string (also using a PRNG) before selecting a "random" character was deemed necessary or possibly "more random." :smile: